daemon/policy.go

```diff
@@ -210,8 +211,24 @@ func (h *getPolicyResolve) Handle(params GetPolicyResolveParams) middleware.Resp
 
 	// Return allowed verdict if policy enforcement isn't enabled between the two sets of labels.
 	if !isPolicyEnforcementEnabled {
+		buffer := new(bytes.Buffer)
+		ctx := params.IdentityContext
+		searchCtx := policy.SearchContext{
+			From:    labels.NewSelectLabelArrayFromModel(ctx.From),
+			Trace:   policy.TRACE_ENABLED,
+			To:      labels.NewSelectLabelArrayFromModel(ctx.To),
+			DPorts:  ctx.Dports,
+			Logging: logging.NewLogBackend(buffer, "", 0),
+		}
+		if ctx.Verbose {
+			searchCtx.Trace = policy.TRACE_VERBOSE
+		}
+		verdict := api.Allowed.String()
+		searchCtx.PolicyTrace("Result: %s\n", strings.ToUpper(verdict))
+		msg := fmt.Sprintf("%s\n  Policy enforcement is not enabled for these two sets of labels\n%s", searchCtx.String(), buffer.String())
```

Comment:
This message is a bit misleading. I assume `isPolicyEnforcement` is reflecting the state of the daemon? I agree that we should report that even if the policy evaluation is DENIED and policy is disabled, then we should indicate this but it must be phrased better. "Enabled for these two sets of labels" is not clear to me.

Reply:
`isPolicyEnforcement` checks if any rules in the policy repository match the provided sets of labels. Since we are determining whether there are rules corresponding to the provided sets of labels, I figured we'd return a statement that corresponds with that. However, I can see why this would be confusing. What do you think would be a better way of phrasing it? I could make the message more specific depending upon which conditions are hit in lines 190-207. Would that be better?

Reply:
Yes, make the explanation dependant on the mode. The message should be very clear if policy enforcement is disabled entirely. The message could also say something along the lines of "policy enforcement enabled (>0 rules loaded).

Reply:
I've added case-specific messages to explain when policy enforcement is not enabled.